### PR TITLE
Pointing at something on yourself now shows the item

### PR DESCRIPTION
**Pull request by Mothblocks**
## About The Pull Request

https://user-images.githubusercontent.com/35135081/180297337-ccffef54-072a-42ab-a56a-4ec95fdebebb.mp4

alpha

image

old-ish, before i moved the arrow:

dreamseeker_2022-07-21T11-20-40

(ignore the hover outline i fixed it)
dreamseeker_2022-07-21T11-05-19

## Why It's Good For The Game
Further reducing reliance on reading the chat box. Previously it wasn't obvious someone pointing at themselves was pointing at something on them, and not just, them.

## Changelog

:cl:
qol: Pointing at something on yourself now shows the item.
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
image

**Comment by Mothblocks** [comment]
🚫 thoughts 🙅‍♀️

**Review by Imaginos16 — APPROVED** [review]
Simple and cute sprite, love this PR!

File: code/modules/point/point.dm [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+	animate(visual, pixel_x = (tile.x - our_tile.x) * world.icon_size + pointed_atom.pixel_x, pixel_y = (tile.y - our_tile.y) * world.icon_size + pointed_atom.pixel_y, time = 1.7, easing = EASE_OUT)
+
+/atom/movable/proc/create_point_bubble(atom/pointed_atom)
+	var/obj/effect/thought_bubble_effect = new
+
+	var/mutable_appearance/thought_bubble = mutable_appearance(
+		'icons/effects/effects.dmi',
+		"thought_bubble",
+		plane = ABOVE_LIGHTING_PLANE,
```
Comment (MrDoomBringer) [inline code/modules/point/point.dm:33]: Will look weird in pitch black rooms
+If it's dark it should be harder to see what you're pointing at
```suggestion
		plane = POINT_PLANE,
```

**Comment by MrStonedOne** [comment]
Too disruptive, you can't allow people to cast an overlay covering a full tile and expect it to not get abused.

Must either have a alpha so you can't hide shit under it or make it so players can disable seeing them with a key combo.

**Review by MrDoomBringer — CHANGES_REQUESTED** [review]
> Simple and cute sprite

Thanks ;)

File: code/modules/point/point.dm (hunk 1 above)
Comment (Mothblocks) [reply code/modules/point/point.dm:33]: i think i tried this and it made the item pitch black but let me try again after i did a bunch of changes

Comment (Mothblocks) [reply code/modules/point/point.dm:33]: it was something else

**Comment by Mothblocks** [comment]
image

**Comment by moocowswag** [comment]
The fact that its between the tiles already kind of prevents blocking vision so I dont see it being abused.

Actually, it can probably cover up the wanted and id symbol from sec huds so you could possibly increase your chances of sec walking by and not noticing a friend of yours is wanted by pointing at your shoe or plushie or something.

**Comment by cacogen** [comment]
neat feature but doesn't make sense that it's a thought bubble

**Comment by MrDoomBringer** [comment]
@cacogen there needs to be a bubble and if it was speech mimes couldn't do it
ss13 is now a garfield comic don't be mad

**Comment by cacogen** [comment]
it could just be a circle without the tail, the pointer gives you all the information you need

**Comment by necromanceranne** [comment]
I seriously doubt it originally would have blocked anything given its offset so much. If anything, someone spamming that shit should be a red flag. It should have click transparency but I have my doubts about practical usage as obfuscation.

**Comment by MrDoomBringer** [comment]
I'm with necro on this, it looks kinda lame being half-transparent. I'm for the keybind to disable them (which will almost never be used anyway)

**Comment by Mothblocks** [comment]
It looks worse I agree but the code to make a keybind to hide everything work would require annoying plane master stuff that I really don't want to do especially for a keybind that would absolutely not be used

I wanted to make something where if you hovered over it it would fade out but BYOND makes this annoying

File: .vscode/settings.json [hunk 2]
```diff
@@ -40,5 +40,8 @@
 		"editor.defaultFormatter": "esbenp.prettier-vscode",
 		"editor.formatOnSave": true
 	},
+	"workbench.editorAssociations": {
+		"*.dmi": "dmiEditor.dmiEditor"
```
Comment (ShizCalev) [inline .vscode/settings.json:44]: huh

Comment (Mothblocks) [reply .vscode/settings.json:44]: It's an extension by @AnturK and not having this was causing me hard to search issues where it was just opening the DMI as an image rather than through the extension. AnturK said putting it here in .vscode/settings.json to stop this footgun in the future was okay, just happened to be in this PR
